packages/babel-plugin/src/shared/messages.js

```diff
@@ -12,63 +12,42 @@
 // This file contains constants to be used within Error messages.
 // The URLs within will eventually be replaced by links to the documentation website for Stylex.
 
-export const ILLEGAL_ARGUMENT_LENGTH =
-  'stylex.create() should have 1 argument.';
-export const ILLEGAL_ARG_LENGTH_FOR_KEYFRAMES =
-  'stylex.keyframes() should have 1 argument.';
-export const NON_STATIC_VALUE =
-  'Only static values are allowed inside of a stylex.create() call.';
-export const NON_STATIC_KEYFRAME_VALUE =
-  'Only static values are allowed inside of a stylex.keyframes() call.';
+export const illegalArgumentLength = (fn: string, argLength: number): string =>
+  `${fn}() should have ${argLength} argument${argLength === 1 ? '' : 's'}.`;
+export const nonStaticValue = (fn: string): string =>
+  `Only static values are allowed inside of a ${fn}() call.`;
+export const nonExportNamedDeclaration = (fn: string): string =>
+  `The return value of ${fn}() must be bound to a named export.`;
+export const unboundCallValue = (fn: string): string =>
+  `${fn}() calls must be bound to a bare variable.`;
+export const nonStyleObject = (fn: string): string =>
+  `${fn}() can only accept a style object.`;
+
 export const ESCAPED_STYLEX_VALUE =
   'Escaping a stylex.create() value is not allowed.';
```

Comment:
Let's remove all the `stylex.` prefixes from the other messages, as done in https://github.com/facebook/stylex/pull/1019/files#diff-37d9716b1d4880fca33aae3ff3c7ffa046c842e27f748609a01684ff5e758bb4